views/incorrect-information/wrong-officers.html

```diff
@@ -0,0 +1,112 @@
+                  <div class="app-step-nav__header" data-position="1">
+                    <h2 class="app-step-nav__title">
+                      <span class="app-step-nav__circle app-step-nav__circle--number">
+                        <span class="app-step-nav__circle-inner">
+                          <span class="app-step-nav__circle-background">
+                            <span class="govuk-visually-hidden">Step</span> 1
+                          </span>
+                        </span>
+                      </span>
+                      <span class="js-step-title">
+                        Update the director details
+                      </span>
+                      </h2>
+                      </div>
+
+                      <div>
+                        <ol class="app-step-nav__list " data-length="1">
+                          <li class="app-step-nav__list-item js-list-item ">
+                            <p>You can update these details online using the relevant form on our
+                              <a href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" target="_blank" rel="noopener noreferrer">WebFiling service (opens in a new tab)</a>.</p>
+                          </li>
+                        </ol>
+                      </div>
+
+                <li class="app-step-nav__step js-step" id="get-a-provisional-licence">
+                  <div class="app-step-nav__header js-toggle-panel" data-position="2">
+                    <h2 class="app-step-nav__title">
+                      <span class="app-step-nav__circle app-step-nav__circle--number">
+                        <span class="app-step-nav__circle-inner">
+                          <span class="app-step-nav__circle-background">
+                            <span class="govuk-visually-hidden">Step</span> 2
+                          </span>
+                        </span>
+                      </span>
+                      <span class="js-step-title">
+                        Return to this service
+                      </span></h2></div>
+
+                  <div>
+                    <ol class="app-step-nav__list " data-length="1">
+                      <li class="app-step-nav__list-item js-list-item ">
+                        <p>Close the WebFiling tab on your device to return to this screen.</p>
+                        <p>If this session has timed out, you will need to sign back in to the <a href="/confirmation-statement">File a confirmation statement service</a>.</p>
+                      </li>
+                    </ol>
+                  </div>
+
+                </li>
+
+                <li class="app-step-nav__step js-step" id="driving-lessons-and-practice">
+                  <div class="app-step-nav__header js-toggle-panel" data-position="3">
+                    <h2 class="app-step-nav__title">
+                      <span class="app-step-nav__circle app-step-nav__circle--number">
```

Comment:
for the duplicated blocks highlighted in sonar, could you make the into 'includes' and then you can include them here and in wrong-psc-details.html